app/routes.js

```diff
@@ -9,9 +9,48 @@ import {
 const router = express.Router();
 
 router.get('/', async (req, res) => {
-
   res.render('home-page', {});
 });
+router.get('/view-solution/:solutionId', async (req, res) => {
+  const context = {
```

Comment:
I assume you are going to get this data from the API in a different PR?

Reply:
was going to ... but I've linked it up now :)